src/components/app-bar/BurgerMenu.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+	import {
+		activeAccount,
+		activeAccountBounties,
+		walletConnect as wcConnection
+	} from '../../stores';
+	import { onMount } from 'svelte';
+
+	let burgerMenuOpen = false;
+	let burgerMenuContainer: HTMLDivElement | null = null;
```

Comment:
```suggestion
	let open = false;
	let container: HTMLDivElement | null = null;
```
Since the whole code is inside BurgerMenu component, we don’t need to add burger and menu to any of the names in the file